TRAMS-API/Models/D365/GetAcademyTransfersProjectsD365Model.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace API.Models.D365
+{
+    public class GetAcademyTransfersProjectsD365Model : BaseD365Model
+    {
+        [JsonProperty("sip_academytransfersprojectid")]
+        public Guid ProjectId { get; set; }
+
+        [JsonProperty("sip_projectname")]
+        public string ProjectName { get; set; }
+
+        [JsonProperty("sip_projectinitiatorfullname")]
+        public string ProjectInitiatorFullName { get; set; }
+
+        [JsonProperty("sip_projectinitiatoruniqueid")]
+        public string ProjectInitiatorUid { get; set; }
+
+        [JsonProperty("sip_projectstatus")]
+        public int ProjectStatus { get; set; }
```

Comment:
Suggestion: Use an enum if these are likely to be static values.

Reply:
Will fix in PR addressed to GET Projects and GET Projects/{id}